src/lib.rs

```diff
@@ -128,6 +132,20 @@ pub fn collect_stars(config: Config) -> Result<(), Box<dyn error::Error>> {
     }
     println!("Collected {} stars", stars.len());
 
+    let path = Path::new("output.md");
+    let display = path.display();
+
+    let mut file = match File::create(&path) {
+        Err(why) => panic!("couldn't create {}: {}", 
+                            display, 
+                            why.description()),
```

Comment:
I had to go looking to find the `description()` method, I don't think I've ever actually seen it before!

It's defined on the `std::error::Error` trait, and is apparently [soft-deprecated](https://doc.rust-lang.org/std/error/trait.Error.html#method.description) and not recommended for use.